wsmatrix@martin.zurowietz.de/overview/controlsManagerLayout.js

```diff
@@ -9,21 +9,23 @@ var ControlsManagerLayout = class {
     constructor() {
         this.originalLayout = null;
         this._overrideProperties = {
-            _computeWorkspacesBoxForState(state, workAreaBox, searchHeight, dashHeight, thumbnailsHeight) {
```

Comment:
This is an incompatible change. @mzur, would you please open a shell-42 branch (or take this as it)?